app/ml_models/rnn/data_tools.py

```diff
@@ -1,34 +1,46 @@
-import torch
 import os
+from typing import List
+
+import torch
 from torch.utils.data import Dataset
 
+from app.ml_models.rnn.vocabulary import Vocabulary
+
 
 class WordLevelDataset(Dataset):
     def __init__(
         self,
         prefix: str = "../../../data/",
-        filename_dataset: str = "straattaal.txt",
+        filename_datasets: List[str] = ["straattaal.txt"],
+        vocabulary: Vocabulary = None,
         filename_vocab: str = "vocabulary.txt",
     ):
-        """Initialize a WordLevelDataset object.
+        """
+        Initialize a WordLevelDataset object and accompanying Vocabulary object if necessary.
```

Comment:
Nit, officially google docstring is this if I'm not mistaken though I might as well be:
```suggestion
        """Initialize a WordLevelDataset object and accompanying Vocabulary object if necessary.
```